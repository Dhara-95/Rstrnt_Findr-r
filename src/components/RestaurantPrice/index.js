import React from "react";

import { checkValidPriceRating } from "../../utils/checkValidPriceRating";

const RestaurantPrice = (props) => {
  const { price } = props;

  return <h2 className="text-grey">{checkValidPriceRating(price)}</h2>;
};

export { RestaurantPrice };
