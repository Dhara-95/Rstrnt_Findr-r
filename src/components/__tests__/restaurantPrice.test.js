import React from "react";
import renderer from "react-test-renderer";

import { RestaurantPrice } from "../RestaurantPrice";

it("displays the Restaurants Price rating correctly", () => {
  const tree = renderer.create(<RestaurantPrice price={"\u00a3"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
