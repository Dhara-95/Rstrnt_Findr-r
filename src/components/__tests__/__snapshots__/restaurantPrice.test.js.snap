// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Restaurants Price rating correctly 1`] = `
<h2
  className=" restaurant-price text-grey"
>
  Â£
</h2>
`;
